deploy/script/katello-restore

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-restore <backup directory>"
+  echo " eg: $ katello-restore /var/tmp/katello-backup-201501261543"
```

Comment:
Would be nice if the suggested restore and suggested backup locations were the same between the two scripts.